chainer/links/model/vision/resnet.py

```diff
@@ -545,15 +546,22 @@ class BuildingBlock(link.Chain):
             (Facebook ResNet).
     """
 
-    def __init__(self, n_layer, in_channels, mid_channels,
-                 out_channels, stride, initialW=None, downsample_fb=False):
+    def __init__(self, n_layers=None, in_channels=None, mid_channels=None,
+                 out_channels=None, stride=None, initialW=None,
+                 downsample_fb=None, n_layer=None):
```

Comment:
Instead of explicitly adding a redundant parameter, code would be nicer using `**kwargs`
Thanks!

Reply:
I revised my PR to use `**kwargs`. :pray: